web/modules/custom/rdf_demo/src/Controller/RdfDemoController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @file
+ * Starting point for integrating ADMS.
+ */
+
+namespace Drupal\rdf_demo\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * Show a list of federated repositories.
+ */
+class RdfDemoController extends ControllerBase {
+
+  /**
+   * Returns a simple page.
+   *
+   * @return array
+   *   A simple render array.
+   */
+  public function repositories() {
+    /** @var \Drupal\rdf_entity\Entity\RdfEntitySparqlStorage $entity_storage */
+    $entity_storage = \Drupal::service('entity.manager')->getStorage('rdf_entity');
+    $query = $entity_storage->getQuery()
+      ->sort('id')
+      ->condition('?entity', 'rdf:type', '<http://www.w3.org/ns/adms#AssetRepository>')
+      ->pager(10);
+
+    $rids = $query->execute();
+    $entities = $entity_storage->loadMultiple($rids);
+    $list = array('#theme' => 'item_list');
+    /** @var Rdf $entity */
+    foreach ($entities as $entity) {
+      $list['#items'][] = array('#markup' => $entity->link());
```

Comment:
Interesting to see that you can just put a link object in `#markup` and it will be rendered correctly. It probably has a magic `__toString()` taking care of this.